tests/infrastructure/test_data_manager_stack.py

```diff
@@ -1,23 +1,29 @@
-from aws_cdk.assertions import Template
+import pytest
+from aws_cdk.assertions import Match, Template
 
 from sds_data_manager.sds_data_manager_stack import SdsDataManagerStack
 
 
-def test_sds_data_manager_stack(app, sds_id):
+@pytest.fixture(scope="module")
+def template(app, sds_id):
     stack_name = f"stack-{sds_id}"
     stack = SdsDataManagerStack(app, stack_name, sds_id)
     template = Template.from_stack(stack)
+    return template
 
+
+def test_s3_data_bucket_resource_count(template):
     template.resource_count_is("AWS::S3::Bucket", 1)
-    # Delete and update are outside of the Properties section
```

Comment:
I thought we had two s3 buckets, but I guess you haven't merged your other PR yet. So you'll likely need to update this again once that other PR gets merged. We should try and get this in sooner rather than later so you aren't having to do all of the rebasing/updating.

Reply:
Agreed. I only added a couple tests for the s3 config PR so this shouldn't be a big deal to resolve merge conflicts